docker-compose.yml

```diff
@@ -882,6 +888,32 @@ services:
 
   ############################### Python ######################################
 
+  alpine-linux-python:
+    # Usage:
+    #   docker compose build alpine-linux-cpp
+    #   docker compose build alpine-linux-python
+    #   docker compose run --rm alpine-python
+    # Parameters:
+    #   ALPINE_LINUX: 3.18
+    #   ARCH: amd64, arm64v8, ...
+    image: ${REPO}:${ARCH}-alpine-linux-${ALPINE_LINUX}-python-3
+    build:
+      context: .
+      dockerfile: ci/docker/linux-apt-python-3.dockerfile
+      cache_from:
+        - ${REPO}:${ARCH}-alpine-linux-${ALPINE_LINUX}-python-3
+      args:
+        base: ${REPO}:${ARCH}-alpine-linux-${ALPINE_LINUX}-cpp
+    shm_size: *shm-size
+    environment:
+      <<: [ *common, *ccache ]
+    volumes: *alpine-linux-volumes
+    command: &python-command >
+      /bin/bash -c "
+        /arrow/ci/scripts/cpp_build.sh /arrow /build &&
+        /arrow/ci/scripts/python_build.sh /arrow /build &&
+        /arrow/ci/scripts/python_test.sh /arrow"
+
```

Comment:
Can we remove this?
We don't need this for musllinux wheel, right?

Reply:
Yes, this is a leftover from when I was trying to add musllinux wheels. I'll remove this.